<launch>
  <node name="pentapod_engine_node" 			pkg="pentapod_engine"  type="pentapod_engine_node" output="screen">
	  <param name="cortex_serial_port"         	type="string" value="/dev/ttyS1"/>
	  <param name="cortex_serial_baudrate"     	type="int"    value="230400"/>
	  <param name="cortex_i2c_port"         	type="string" value="/dev/i2c-1"/>
	  <param name="cortex_i2c_address"     		type="int"    value="08"/>
  </node>
</launch>

